contracts/services/neutron-ibc-transfer/src/msg.rs

```diff
@@ -0,0 +1,106 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Deps, DepsMut, Uint128, Uint64};
+use cw_ownable::cw_ownable_query;
+use getset::{Getters, Setters};
+use neutron_sdk::bindings::query::NeutronQuery;
+use valence_macros::OptionalStruct;
+use valence_service_utils::{
+    denoms::{CheckedDenom, UncheckedDenom},
+    error::ServiceError,
+    msg::ServiceConfigValidation,
+    ServiceAccountType, ServiceConfigInterface,
+};
+
+#[cw_serde]
+pub enum ActionsMsgs {
+    IbcTransfer {},
+}
```

Comment:
Can we rename to ActionMsgs? (Some services had ActionMsgs and others ActionMsgs so I renamed them all to ActionMsg), this way we can keep it consistent everywhere